static/js-src/cs-client.js

```diff
@@ -256,8 +256,12 @@ class ChromeStatusClient {
       (resp) => resp['features_by_type']);
   }
 
-  searchFeatures(userQuery) {
-    return this.doGet(`/features?q=${userQuery}`);
+  searchFeatures(userQuery, sortSpec) {
+    let url = `/features?q=${userQuery}`;
+    if (sortSpec) {
+      url += '&sort=' + sortSpec;
+    }
```

Comment:
This functionality has no visibility outside of the query string, right? Maybe it would be useful to define a default sort that can be seen on the query string, if only for a little additional visibility.

Reply:
These URLs are never seen by the user anyway because they are API calls.   

I think the way to let the user see the sort order would be in a UI element that is near the search filter widgets.  However, I was thinking that that would only be shown on the feature list page.  On the feature list page, sort order would need to be in the URL so that people can bookmark it and share it.  For the my features page, I'd rather keep the page pretty streamlined, without many options cluttering the page.  If users ask for the ability to customize their my features page, we could take that on at that time.

Reply:
I see. After looking again, I'm seeing that this is a change to allow receiving the data from the API in a sorted order and doesn't necessary add any user functionality besides having default sort orders for the reviews and approvals section now. Makes sense!